packages/solana/src/classes/deployer.ts

```diff
@@ -100,4 +105,18 @@ export class Deployer {
       .run();
     return collectionNft.mint.address.toBase58();
   }
+
+  async createNftDrop(metadata: NFTDropMetadataInput): Promise<string> {
+    invariant(this.wallet.signer, "Wallet is not connected");
+    const parsed = NFTDropContractSchema.parse(metadata);
+
+    // TODO create a collection with metadata and associate it with the drop
```

Comment:
Still need to do this right?

Reply:
Yes - want to do this in a separate PR

Reply:
I think this should go in a separate PR. Experimented with it a bit and its not as straightforward + doesn't seem obvious to me yet that we should do this for every drop (I want to look more into how the candy machine works internally/associates all its NFTs together already).